src/RileyMaze/toolbox.js

```diff
@@ -47,9 +51,9 @@ var RMZToolbox = GObject.registerClass(class RMZToolbox extends Toolbox {
 
     bindWindow(win) {
         win.get_style_context().add_class('RileyMaze');
-        this._unitTopic.bindWindow(win);
```

Comment:
It looks like `RMZUnit.bindWindow()` is obsolete, so you should go ahead and remove that method as well. Additionally it seems like it might be better to do that in a separate commit since at first glance it looks like it's fixing a separate bug.